fem/fe.cpp

```diff
@@ -2550,6 +2550,741 @@ void TriLinear3DFiniteElement::CalcDShape(const IntegrationPoint &ip,
    dshape(7,2) =   ox *  y;
 }
 
+BiLinear3DFiniteElement::BiLinear3DFiniteElement()
```

Comment:
@v-dobrev -- do you think we should keep supporting these, or should we focus on the `H1_*` elements for the future?